src/dailyai/services/daily_transport_service.py

```diff
@@ -225,6 +229,16 @@ def _post_run(self):
         self.client.leave()
         self.client.release()
 
+    def _handle_video_frame(self, participant_id, video_frame):
+        # TODO-CB: What about multiple participants?
```

Comment:
Just saw this comment... I think you would need a `receive_queue` per participant.